docs/docs/creating-a-source-plugin.md

```diff
@@ -13,16 +13,16 @@ Once a source plugin brings data into Gatsby's system, it can be transformed fur
 At a high-level, a source plugin:
 
 - Ensures local data is synced with its source and is 100% accurate.
-- Creates nodes with accurate media types, human meaningful types, and accurate
-  contentDigests.
+- Creates [nodes](/docs/node-interface/) with accurate media types, human-readable types, and accurate
+  [contentDigests](/docs/node-interface/#contentdigest).
 - Links nodes & creates relationships between them.
 - Lets Gatsby know when nodes are finished sourcing so it can move on to processing them.
 
 ## What does the code look like?
 
-A source plugin is a normal NPM package. It has a package.json with optional
-dependencies as well as a `gatsby-node.js` where you implement Gatsby's Node.js
-APIs.
+A source plugin is a regular NPM package. It has a `package.json` file with optional
```

Comment:
If we want to make things super accurate we'd also need to mention here that a plugin also needs a `main` key in its `package.json` file that points to a `index.js` file that can export components/files or just be empty with a `//noop`. Alternatively this `main` field would point to the mentioned `gatsby-node.js` file.

Reply:
This is probably ok to leave out from this section? As it's covered in the linked _Files Gatsby Looks for in a Plugin_.

Reply:
Ah, forgot that it was added there. Scratch that comment then :)